samples/client/echo_api/java/native/src/test/java/org/openapitools/client/CustomTest.java

```diff
@@ -245,4 +246,25 @@ public void testDefaultValuesSerializationWithIncorrectDefaultValues() throws IO
                     " at [Source: (String)\"{ \"array_string_enum_default\": [\"invalid\"] }\"; line: 1, column: 33] (through reference chain: org.openapitools.client.model.DefaultValue[\"array_string_enum_default\"]->java.util.ArrayList[0])");
         }
     }
+
+
+    /**
+     * Test form parameter(s)
+     *
+     * Test form parameter(s)
+     *
+     * @throws ApiException
+     *          if the Api call fails
+     */
+    @Test
+    public void testFormIntegerBooleanStringTest() throws ApiException {
+        Integer integerForm = 1337;
+        Boolean booleanForm = true;
+        String stringForm = "Hello World";
+        String response = formApi.testFormIntegerBooleanString(integerForm, booleanForm, stringForm);
+
+        org.openapitools.client.EchoServerResponseParser p = new org.openapitools.client.EchoServerResponseParser(response);
+        Assert.assertEquals("/form/integer/boolean/string", p.path);
+        Assert.assertTrue(p.body.contains("Hello%20World"));
```

Comment:
one last suggestion: can we do an exact match using assertEquals instead?

We want to catch any potential change in the body